kotlin/display-grid/src/main/java/com/esri/arcgisruntime/sample/displaygrid/MainActivity.kt

```diff
@@ -0,0 +1,299 @@
+/* Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.esri.arcgisruntime.sample.displaygrid
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.View
+import android.widget.AdapterView
+import android.widget.AdapterView.OnItemSelectedListener
+import android.widget.ArrayAdapter
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.geometry.Point
+import com.esri.arcgisruntime.geometry.SpatialReference
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.Viewpoint
+import com.esri.arcgisruntime.mapping.view.Grid
+import com.esri.arcgisruntime.mapping.view.LatitudeLongitudeGrid
+import com.esri.arcgisruntime.mapping.view.MgrsGrid
+import com.esri.arcgisruntime.mapping.view.UsngGrid
+import com.esri.arcgisruntime.mapping.view.UtmGrid
+import com.esri.arcgisruntime.symbology.LineSymbol
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol
+import com.esri.arcgisruntime.symbology.TextSymbol
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.popup_menu.view.*
```

Comment:
import the whole popup_menu so you don't need to inflate it.
```suggestion
import kotlinx.android.synthetic.main.popup_menu.*
```